src/components/AdvancedOptions.tsx

```diff
@@ -68,12 +69,16 @@ export default function AdvancedOptions(): ReactElement {
       </span>
       <div className={styles.divider}></div>
       <div
+        dat-testid="tableContainer"
         className={classNames(styles.tableContainer, {
           [styles.show]: advanced,
         })}
       >
         <div className={styles.parameter}>
-          <div className={styles.infiniteApproval}>
+          <div
+            data-testid="infiniteApproval"
```

Comment:
Good name here, explicit. Maybe even naming it `infiniteApprovalContainer` would make it more specific.